integrations/tensorflow/bindings/python/iree/tf/support/tf_benchmark_utils.py

```diff
@@ -0,0 +1,240 @@
+# Lint as: python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Benchmark utilities interop with TensorFlow."""
+
+# pylint: disable=missing-docstring
+# pylint: disable=protected-access
+# pylint: disable=unsupported-assignment-operation
+
+# TODO(#4131) python>=3.7: Use postponed type annotations.
+
+import collections
+import os
+import tempfile
+from typing import Sequence, Set, Type
+
+from absl import flags
+from absl import logging
+from iree.tf.support import module_utils
+import tensorflow.compat.v2 as tf
+
+flags.DEFINE_string("target_backend", None,
+                    "The target backend to benchmark against.")
+flags.DEFINE_list("configuration_names", None,
+                  "A comma-separated list of configuration names")
+flags.DEFINE_list(
+    "compilation_flags", None,
+    "A comma-separated list of semicolon-separated compilation flags for "
+    "each configuration")
```

Comment:
That's a pretty gross CLI... Parallel lists with semicolon as the minor delimiter and comma as the major (reversed from normal)

Reply:
Actually, I don't need this two levels of nesting. (It was there from the start b/c I was trying to make that work and then found it's not possible at least right now.) So cleaned up a bit.